server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -718,6 +718,68 @@ In order to do that: `default=true`
 
 The web-admin server will return `404` response code when the default identity can not be found.
 
+=== Creating a user identity
+
+API to creating a new identity
```

Comment:
```suggestion
API to create a new JMAP user-set identity
```
Maybe? 
The idea here is to distinguish with server-set identity which I guess we can not really create them with this API.